CHANGELOG.md

```diff
@@ -4,8 +4,8 @@
 
 ### Adds
 
-TODO: set documentation link once created:
-* Adds the widget placeholders feature. Checkout the [Widget Placeholders documentation](https://v3.docs.apostrophecms.org/) for more detail. 
+* Adds the widget placeholders feature to the core and custom widgets. This includes toggling placeholder content, replacing default content for core widgets, and adding placeholder content for multiple schema fields.
```

Comment:
```suggestion
* Adds the widget placeholders feature to the core and custom widgets. This includes toggling placeholder content when adding widgets to an area, replacing default content for core widgets, and adding placeholder content for multiple schema fields.
```

Something like that, to emphasize the fact they placeholders are there when adding the widgets,  what do you think?